tools/cldr-apps/src/test/java/org/unicode/cldr/unittest/web/TestSTFactory.java

```diff
@@ -451,6 +451,11 @@ public void TestVettingDataDriven() throws SQLException, IOException {
         runDataDrivenTest(TestSTFactory.class.getSimpleName()); // TestSTFactory.xml
     }
 
+    public void TestVotePerf() throws SQLException, IOException {
+        if (TestAll.skipIfNoDb()) return;
+        runDataDrivenTest("TestVotePerf"); // TestSTFactory.xml
```

Comment:
TestVotePerf.xml, not TestSTFactory.xml?

Reply:
New file

Reply:
the comment however shows the name of the old file -- if that's intentional, I don't get it!

Reply:
```suggestion
        runDataDrivenTest("TestVotePerf"); // TestVotePerf.xml
```

Reply:
you are right!